<ui:component xmlns="http://www.w3.org/1999/xhtml"
	xmlns:h="http://xmlns.jcp.org/jsf/html"
	xmlns:f="http://xmlns.jcp.org/jsf/core"
	xmlns:ui="http://xmlns.jcp.org/jsf/facelets"
	xmlns:p="http://primefaces.org/ui" 
	xmlns:jsf="http://xmlns.jcp.org/jsf"
	xmlns:pt="http://xmlns.jcp.org/jsf/passthrough"
	xmlns:fn="http://java.sun.com/jsp/jstl/functions"
	xmlns:pe="http://primefaces.org/ui/extensions">
	
	<p:ajaxExceptionHandler type="br.gov.acrimat.exception.BeanException" update="dialogException"
		onexception="PF('dialogException').show()"/>
	
	<p:outputPanel styleClass="dialogContainer"/>
	<p:dialog dynamic="true" draggable="true" id="dialogException"
		widgetVar="dialogException" resizable="false" 
		closeOnEscape="true" header="Exception" responsive="true"
		onShow="$('.dialogContainer').show()" styleClass="dialogException"
		onHide="$('.dialogContainer').hide();">
		<h:outputLabel value="Ocorreu um erro inesperado. Tente novamente." />
	</p:dialog>
</ui:component>